miniwin/sdl3gpu/src/miniwin_ddraw.cpp

```diff
@@ -54,11 +54,11 @@ HRESULT DirectDrawImpl::CreatePalette(
 		return DDERR_INVALIDPARAMS;
 	}
 
-	*lplpDDPalette = static_cast<LPDIRECTDRAWPALETTE>(new DirectDrawPaletteImpl(lpColorTable));
+	*lplpDDPalette = static_cast<LPDIRECTDRAWPALETTE>(new DirectDrawPalette_SDL3GPUImpl(lpColorTable));
 	return DD_OK;
 }
 
-HRESULT DirectDrawImpl::CreateSurface(
+HRESULT DirectDraw_SDL3GPUImpl::CreateSurface(
```

Comment:
This names are just really misleading, it's a SDL_Surface implementation and the goal is to move it to SDL_Render / SDL_Texture.

Reply:
True. Since this class is member of the sdl3gpu backend, I did a simple `s/Impl/_SDL3GPUIml/` substitution.
It made sense for all classes of a backend to have the same suffix.

This is just naming. As long as we don't have name clashes.